api/envoy/extensions/filters/http/oauth2/v3alpha/oauth.proto

```diff
@@ -76,6 +76,10 @@ message OAuth2Config {
   // defaults to "user" scope.
   // OAuth RFC https://tools.ietf.org/html/rfc6749#section-3.3
   repeated string auth_scopes = 9;
+
+  // Optional resource parameter for authorization request
+  // RFC: https://tools.ietf.org/html/rfc8707
+  string resource = 10;
```

Comment:
The RFC seems to allow multiple values here, should this be a `repeated` field?

```
Multiple "resource" parameters MAY be
      used to indicate that the requested token is intended to be used
      at multiple resources.
```

Reply:
@snowp I will convert this to a `repeated` field.

Reply:
@snowp Its up for review.